tiledb/ml/models/sklearn.py

```diff
@@ -70,31 +81,12 @@ def preview(self, *, display: str = "text") -> str:
         else:
             return ""
 
-    def _create_array(self) -> None:
+    def _create_array_internal(self) -> None:
```

Comment:
There is no need for separate `_create_array_internal`, it can be merged with `_create_array` and overriden in subclasses. 

Same comment for all `TileDBArtifact` subclasses.